src/ripple/rpc/handlers/AccountTx.cpp

```diff
@@ -240,6 +240,31 @@ doAccountTxHelp(RPC::Context& context, AccountTxArgs const& args)
 
     result.ledgerRange = std::get<LedgerRange>(lgrRange);
 
+    // if the account does not exist in any of the ledger range,
+    // return an error
+    if (context.apiVersion > 1u)
+    {
+        bool accountFound = false;
+        for (unsigned int idx = result.ledgerRange.min;
+             idx <= result.ledgerRange.max;
+             ++idx)
```

Comment:
Is iterating through the range efficient here? Would there be a more efficient approach?

cc. @ximinez

Reply:
* If the account exists at the ledger range start, it'll be found right away. :heavy_check_mark: 
* If the account is created sometime inside the range, [it can't be deleted for at least 255 ledgers](https://github.com/XRPLF/rippled/blob/c197c4cdd97612b5c3b71b3919add5238fae93a6/src/ripple/app/tx/impl/DeleteAccount.cpp#L213-215). That means that you don't need to check every single ledger. You can check every 255. However, we want to be sure to check the last ledger in case the account is created there. We don't want to check outside the end of the range in case that ledger doesn't even exist yet. :x: 
  * Example: We're looking at ledgers 1-600 for an account. If the account is created in ledger 2, and doesn't send any transactions of its own, then to be deleted `(2 + 255 <= view.seq())` must be true. That happens at ledger 257. So if you iterate every 255 ledgers, you'll start by checking 1, and next check 256, so the account will be there. (I'm choosing to iterate every 255 because I don't think the account will exist in ledger 257 if it's deleted there, and we don't want to miss it).  Now consider if the account is created in ledger 512. You'll check 1, 256, and 511. The next check would be 766, which is out of our range. If the last validated ledger is 605, you won't find the ledger, and that would be bad, so we want to make sure to explicitly check ledger 600 before exiting the loop. The account can't be deleted until `512+255 == 767`, so it'll be there.

Here's what I'm thinking:
1. First refactor AccountDelete.cpp to move `seqDelta` to src/ripple/protocol/SystemParameters.h and call it something like `AccountDeleteSeqDelta`.
2. Change the loop as such:
```
unsigned int const rangeEnd = result.ledgerRange.max + AccountDeleteSeqDelta;
for (unsigned int idx = result.ledgerRange.min;
    idx < rangeEnd;
   idx += AccountDeleteSeqDelta)
{
  if (idx > result.ledgerRange.max)
   idx = result.ledgerRange.max;
 [...]
```
I think I got that right.

Reply:
@ximinez if the user wants to query the max ledger range. Wouldnt it still be inefficient? Iterating at an increment of 255 will still need over 300k iterations for the max ledger range. 

I am curious if the trade off is really worth it to implement an acctNotFound error. IMO we should stick with the current behavior.

Reply:
@shawnxie999 You're talking about full history? That's a good point. Maybe this check should only run if the difference between `result.ledgerRange.min` and `max` is below some reasonable limit (255*10?).

I am not so sure if this trade off of time and code complexity is worth it either.

Reply:
@ximinez @shawnxie999  Thanks for the convo! I never knew the ledger range was over 300k iterations....
If that is the case, I agree that the trade off of time and code complexity is definitely not worth the change, and perhaps the docs should be updated instead.